source/refined-github.css

```diff
@@ -27,13 +27,7 @@ https://github.com/notifications (Grouped by repo)
 https://github.com/notifications?query=reason%3Acomment (which is an unsaved filter)
 */
 .js-check-all-container .js-bulk-action-toasts ~ div .Box-header {
-	height: auto !important;
 	flex-wrap: wrap;
```

Comment:
I'm considering dropping this and restore the old width on mobile too. The same issue still happens on mobile because the buttons actually wraps into two lines.

I think the dropdown was an elegant solution to the issue, but I prefer that the header just scrolls instead, so this issue can be solved without wrapping:

- https://github.com/refined-github/refined-github/issues/5704